hexrd/ui/calibration/polar_plot.py

```diff
@@ -151,3 +152,21 @@ def write_image(self, filename='polar_image.npz'):
             with h5py.File(filename, 'w') as f:
                 for key, value in data.items():
                     f.create_dataset(key, data=value)
+
+    def write_maud(self, filename='polar_to_maud.esg'):
+        filename = Path(filename)
+
+        with open(filename, 'wb') as fid:
+            eta_vec = np.degrees(self.angular_grid[0])
+            for i, eta in enumerate(np.average(eta_vec, axis=1).flatten()):
+                if i == 0:
+                    hstr = header0 % (i, np.linalg.norm(self.pv.tvec_s), eta)
+                else:
+                    hstr = header % (i, eta)
+                fid.write(hstr.encode('ascii'))
+                fid.write(block_hdr.encode('ascii'))
+                tth = HexrdConfig().last_unscaled_azimuthal_integral_data[0]
+                intensities = self.raw_rescaled_img
```

Comment:
Since `self.raw_rescaled_img` is a computed property, can you move this line outside of the loop?

Reply:
Yes, thank you!